lib/cpp/src/thrift/TApplicationException.h

```diff
@@ -21,6 +21,7 @@
 #define _THRIFT_TAPPLICATIONEXCEPTION_H_ 1
 
 #include <thrift/Thrift.h>
+#include <stdint.h>
```

Comment:
I see this work was done in windows/config.h so we just need to have TApplicationException.h include the correct header to pull that in on Windows.